doc/pingdom.md

```diff
@@ -0,0 +1,76 @@
+# Pingdom checks
+
+## Setting up the checks
+
+The checks are defined in terraform configuration file `terraform/pingdom/pingdom.tf`.
+
+### Requirements
+
+* Download or build the terraform-provider-pingdom binary. See below for instructions.
+* Make sure you have access to the PaaS credential store, this is required for Pingdom credentials.
+* Load the AWS credentials for the environment you are setting up checks for. These are required as the Terraform state file for the Pingdom checks is stored in an S3 bucket.
+
+### Usage
+
+#### Configure
+Configure the IDs of the contacts to be notified as a comma-delimited string in variable PINGDOM_CONTACT_IDS in the Makefile.
+
+#### Run
+Run `make <ENV> pingdom` to set up the Pingdom checks.
+
+## Use the binary
+Binaries for Linux and MacOS are available in our [paas-terraform-provider-pingdom repository](https://github.com/alphagov/paas-terraform-provider-pingdom/releases).
```

Comment:
The problem with download automatically is the `.terraformrc` file. I wouldn't like this script to change files in my `$HOME`.

But one solution can be download the binary to a tmp dir, copy the `.terraformrc` there and change the `$PATH` and `$HOME` in the script:

BTW, the binaries should be suffixed with the output of ` $(uname -s)-$(uname -a)`

``` 
mkdir /tmp/terraform-pingdom
wget https://github.com/alphagov/paas-terraform-provider-pingdom/releases/download/0.2.1/terraform-provider-pingdom- $(uname -s)-$(uname -a) -O  /tmp/terraform-pingdom/terraform-provider-pingdom
cp terraform/providers/.terraformrc /tmp/terraform-pingdom/

PATH=$PATH:/tmp/terraform-pingdom HOME=/tmp/terraform-pingdom terraform  destroy  -var pingdom_user=$(paas-pass pingdom.com/username) -var pingdom_password=$(paas-pass pingdom.com/password) -var pingdom_api_key=$(paas-pass pingdom.com/api_key) -var pingdom_account_email=$(paas-pass pingdom.com/account_email) -var apps_dns_zone_name=example.com -var env=hector
```